test_unstructured/partition/test_text.py

```diff
@@ -50,6 +50,15 @@ def test_partition_text_from_file():
     assert elements == EXPECTED_OUTPUT
 
 
+def test_partition_text_from_bytes_file():
+    filename = os.path.join(DIRECTORY, "..", "..", "example-docs", "fake-text.txt")
+    with open(filename, "rb") as f:
+        file_bytes = f.read()
+        elements = partition_text(file=file_bytes)
```

Comment:
why do you pass the bytes data from f.read() rather than directly passing in f here? (since f is the file-like object)

Reply:
So I fixed it to pass the bytes file, but is it also reasonable to assume we could be passed a byte string?? The former is how I discovered the error, but it seems like the latter could also arise? Thoughts?

Reply:
ahh, so sorry, I missed your response from the other day. Yep, definitely agreed on followup ticket, that sounds great!